LayoutTests/imported/w3c/web-platform-tests/scroll-animations/css/scroll-timeline-shorthand-expected.txt

```diff
@@ -1,52 +1,52 @@
-FAIL Shorthand contraction of scroll-timeline-name:--a, --b:undefined;scroll-timeline-axis:inline, block:undefined assert_equals: Declared value expected "--a inline, --b" but got ""
-FAIL Shorthand contraction of scroll-timeline-name:none, none:undefined;scroll-timeline-axis:block, block:undefined assert_equals: Declared value expected "none, none" but got ""
-PASS Shorthand contraction of scroll-timeline-name:--a, --b, --c:undefined;scroll-timeline-axis:inline, inline:undefined
-PASS Shorthand contraction of scroll-timeline-name:--a, --b:undefined;scroll-timeline-axis:inline, inline, inline:undefined
+PASS Property scroll-timeline value 'none block'
+PASS Property scroll-timeline value '--abc inline'
+PASS Property scroll-timeline value 'none y'
+PASS Property scroll-timeline value '--abc x'
+PASS Property scroll-timeline value '--y y'
+PASS Property scroll-timeline value '--abc'
+PASS Property scroll-timeline value '--inline block'
+PASS Property scroll-timeline value '--block block'
+PASS Property scroll-timeline value '--y block'
+PASS Property scroll-timeline value '--x block'
+PASS Property scroll-timeline value '--a, --b, --c'
+PASS Property scroll-timeline value '--a inline, --b block, --c y'
+PASS e.style['scroll-timeline'] = "--abc y" should set scroll-timeline-axis
+PASS e.style['scroll-timeline'] = "--abc y" should set scroll-timeline-name
+PASS e.style['scroll-timeline'] = "--abc y" should not set unrelated longhands
+PASS e.style['scroll-timeline'] = "--inline x" should set scroll-timeline-axis
+PASS e.style['scroll-timeline'] = "--inline x" should set scroll-timeline-name
+PASS e.style['scroll-timeline'] = "--inline x" should not set unrelated longhands
+PASS e.style['scroll-timeline'] = "--abc y, --def" should set scroll-timeline-axis
+PASS e.style['scroll-timeline'] = "--abc y, --def" should set scroll-timeline-name
+PASS e.style['scroll-timeline'] = "--abc y, --def" should not set unrelated longhands
+PASS e.style['scroll-timeline'] = "--abc, --def" should set scroll-timeline-axis
+PASS e.style['scroll-timeline'] = "--abc, --def" should set scroll-timeline-name
+PASS e.style['scroll-timeline'] = "--abc, --def" should not set unrelated longhands
+PASS Shorthand contraction of scroll-timeline-name:--abc:undefined;scroll-timeline-axis:inline:undefined
+PASS Shorthand contraction of scroll-timeline-name:--a, --b:undefined;scroll-timeline-axis:inline, block:undefined
+PASS Shorthand contraction of scroll-timeline-name:none, none:undefined;scroll-timeline-axis:block, block:undefined
+FAIL Shorthand contraction of scroll-timeline-name:--a, --b, --c:undefined;scroll-timeline-axis:inline, inline:undefined assert_equals: Declared value expected "" but got "--a inline, --b inline, --c"
+FAIL Shorthand contraction of scroll-timeline-name:--a, --b:undefined;scroll-timeline-axis:inline, inline, inline:undefined assert_equals: Declared value expected "" but got "--a inline, --b inline, none inline"
```

Comment:
The WPT suggests that when the name & axis have different lengths, the shorthand should be an empty string. Seems reasonable behavior if the shorthand can't express that by itself.

Reply:
I think this is wrong, see https://github.com/web-platform-tests/wpt/issues/43336. This will be addressed by a separate patch which will update the WPT and contain any required source change to match the new expectations.